deploy/sample/zfspvcr.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+items:
+- apiVersion: openebs.io/v1alpha1
+  kind: ZFSVolume
+  metadata:
+    creationTimestamp: "2019-08-25T08:02:14Z"
+    generation: 2
```

Comment:
Is this yaml expected to be applied by the users?
Can we remove the auto generated portions?

Reply:
no, as of now user will not apply it. but anyway I will remove auto-generated portions.